test_brain_friendly.py

```diff
@@ -1,8 +1,11 @@
+import pytest
```

Comment:
`import pytest` belongs with `import io`, not with all of our `brainfriendly` imports.